crates/sui-framework-snapshot/manifest.json

```diff
@@ -241,7 +241,7 @@
     ]
   },
   "31": {
-    "git_revision": "ac208a8cb1",
+    "git_revision": "95d2011b0e-dirty",
```

Comment:
dirty? Is this intentional?

Reply:
Yes need, its generative

Reply:
Just FYI this shouldn't be updated. 

The rules governing when/how to update this (because it took me forever and a lot of pain to figure this out):
* A new framework snapshot for a given protocol version is only created when that protocol version hits testnet/mainnet. Once created, it should not be updated further -- any further changes to the framework will need to go into a new protocol version.
* A snapshot should not be created for a protocol version unless/until the framework has hit testnet/mainnet.